js/main.js

```diff
@@ -0,0 +1,314 @@
+const buttons = document.getElementsByClassName("button");
+const digitButtons = document.getElementsByClassName("digitButton");
+const screen = document.getElementById("screen");
+const unblockedKeys = [8, 13, 16, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 114, 187];
```

Comment:
Let's create an object where the property name would be the name of the key on the keyboard and value the number of the key. Example:
```
const unblockedKeys = {
  'Backspace': 8,
  'Entere': 13,
  ...
};
const unblockedKeysCodes = Object.values(unblockedKeys);
```
After this code, you could work with variable `unblockedKeysCodes` as before, but you and other people will understand which keys will be unlocked.